assets/scss/modules/_grid-system.scss

```diff
@@ -0,0 +1,97 @@
+@function columns($columns, $total-columns) {
+  @return percentage($columns / $total-columns);
+}
+
+@media screen and (min-width: 320px) {
+  @for $i from 1 through 12 {
+    .col-sm-#{$i} {
+      max-width: columns($i, 12);
+      flex: 0 0 columns($i, 12);
+    }
+  }
+  .dp-container {
+    max-width: 100%;
+    background: $dp-color-lemonyellow;
```

Comment:
What Is the intent of this class `dp-container`? is it fine having a background color associated?

Reply:
We need a container to handle the widths of the columns and create them. The width of the container is handled with the @media queries.
The background color is only to display the @media queries